docs/reference/services.rst

```diff
@@ -0,0 +1,63 @@
+=================
+MicroOVN Services
+=================
+
+This document provides brief explanation of services provided by MicroOVN. They
+are generally not meant for direct user interaction and should be used only
+if necessary. List and status of these services can be displayed by running:
+
+.. code-block:: none
+
+    snap services microovn
+
+microovn.central
+----------------
+
+.. warning::
+
+    Deprecated: This service will be removed in future release
+
+This is a shortcut service. Starting this service will start and enable
+``microovn.ovn-nb``, ``microovn.ovn-sb`` and ``microovn.ovn-northd``. However
+this service , due to the ``snap`` limitations, is not capable of stopping
+child services and its usage is generally discouraged.
+
+microovn.chassis
+----------------
+
+This service maps directly to ``ovn-controller`` daemon.
+
+microovn.daemon
+---------------
+
+Main MicroOVN service/process that manages rest of the processes. It also
+handles communication with other MicroOVN cluster members and provides an API
+for ``microovn`` client command.
+
+microovn.ovn-nb
+---------------
+
+This service maps directly to ``OVN Northbound`` database/service.
+
+microovn.ovn-northd
+-------------------
+
+This service maps directly to ``ovn-northd`` daemon.
+
+microovn.ovn-sb
+---------------
+
+This service maps directly to ``OVN Southbound`` database/service.
+
+microovn.refresh-expiring-certs
+-------------------------------
+
+This service is a recurring process that runs once a day between ``02:00`` and
+``02:30``. It triggers TLS certification reissue for certificates that are
+nearing the expiration. For more information see ``Certificate lifecycle``
+section in :doc:`/how-to/tls`
```

Comment:
Link directly to that section. See:

https://docs.openstack.org/charm-guide/latest/community/doc-contrib/rst.html#linking-to-a-location-within-a-page-in-the-doc-set